### ISSUE TITLE
libmq-anansi files still present

### ISSUE THREAD

**Issue by nickshanks** [bbcarchdev/twine-anansi-bridge#4]
The libmq-anansi plug-in was recently moved from this repository to Anansi. The file.c and p_anansi-plugin.h files however are still here. Are they still needed here?

If so, should the latter be renamed?

**Comment by nevali** [comment]
No… we should just mark this repository as no longer being used.

**Comment by nickshanks** [comment]
It's not used by RES, but still a potentially useful (if under-maintained) part of Acropolis, right?

**Comment by nevali** [comment]
that's true — although given what it actually does (push change notifications from Anansi into any arbitrary message queue), it should probably be renamed, decoupled from Twine, and merged into the Anansi codebase instead

**Comment by nickshanks** [comment]
OK. Shall I raise a new ticket for that, or do you want to? I assume it's quite a low-priority task.

**Comment by nevali** [comment]
I'll raise one against Anansi for the integration.

For now, then, you're right — let's get rid of the libmq-anansi bits from this project at least.

Very low-priority!

M.

**Comment by nevali** [comment]
Actually, hold on a minute. The files aren't part of `libmq-anansi.so` at all. Sorry, I should have checked before replying.

This code _is_ used, because although `libmq-anansi` does the actual queue integration, this project contains the plug-in for **Twine** which lets it understand how to process `application/x-anansi-url` messages (which are URLs into the Anansi cache), as delivered by `libmq-anansi`.

In the long run, we _could_ dispense with this code, if we made `libmq-anansi` send RDF messages directly (although in order to do that it would have to obtain configuration about the cache somehow), but it seems more complicated than it's worth.

This could be better-explained in the `README`.

**Comment by nickshanks** [comment]
I've read the previous comment twice and still don't fully understand the implications. If you update the README can you elaborate?
